packages/expo-notifications/android/build.gradle

```
@@ -95,4 +95,9 @@ dependencies {
   api 'com.google.firebase:firebase-messaging:21.1.0'
 
   api 'me.leolin:ShortcutBadger:1.1.22@aar'
+
+  testImplementation "androidx.test:core:1.4.0"
+  testImplementation "junit:junit:4.13.2"
+  testImplementation "org.robolectric:robolectric:4.7.3"
+  testImplementation "io.mockk:mockk:1.12.3"
```

The `expo-modules-test-core` includes all of the above dependencies + some utilities. This helps keep test dependency versions consistent across all packages 😉 
```suggestion
  if (project.findProject(':expo-modules-test-core')) {
    testImplementation project(':expo-modules-test-core')
  }
```